android/engine/src/main/java/org/smartregister/fhircore/engine/appfeature/AppFeature.kt

```diff
@@ -26,4 +26,6 @@ sealed class AppFeature(val name: String) {
   object PatientManagement : AppFeature(name = "PatientManagement")
   object HouseholdManagement : AppFeature(name = "HouseholdManagement")
   object DeviceToDeviceSync : AppFeature(name = "DeviceToDeviceSync")
+  object Hiv : AppFeature(name = "Hiv")
+  object TracingAndAppointment : AppFeature(name = "TracingAndAppointment")
```

Comment:
being nitpicky again here :D I think we should also rename this but I'm not sure what to yet...

Reply:
good catch @pld , its was cloned and changes might got reverted, pushing with all patients as expected by default.

Reply:
pushed